packages/serverless/package-lock.json

```diff
@@ -1,4191 +1,8 @@
 {
 	"name": "@instana/serverless",
 	"version": "1.137.2",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```

Comment:
Have you switched to Node 16 locally? (NPM 8)
This looks wrong to me :)

Reply:
Yeah, I got lost in the version switch craziness while testing the storage in different versions.
Will fix it. Thanks!